src/main/kotlin/br/unipar/plano/domain/transportes/model/Transporte.kt

```diff
@@ -0,0 +1,91 @@
+package br.unipar.plano.domain.transportes.model
+
+import br.unipar.plano.domain.carteirinhas.model.Carteirinha
+import br.unipar.plano.domain.centrais.model.IdTransporte
+import java.time.LocalDate
+import javax.persistence.*
+
+enum class StatusTransporte {
+    PENDENTE, APROVADO, CANCELADO;
+}
+
+enum class TipoTransporte(val tipo: String) {
+    AMBULANCIA("Ambulância"),
+    UTI_MOVEL("UTI Móvel"),
+    AEREO("Aéreo");}
+
+@Entity
+class Transporte(
+    @field:EmbeddedId
+    val id: IdTransporte,
+
+    @OneToOne(cascade = [CascadeType.ALL], orphanRemoval = true)
```

Comment:
Pode existir mais de uma solicitação de suporte para uma carteirinha (ex. quando enviado para o endereço errado ou devolvida pelos correios).

```suggestion
    @ManyToOne(cascade = [CascadeType.ALL], orphanRemoval = true)
```